styles/stylesheet.css

```diff
@@ -0,0 +1,510 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+/*Style for header starts here*/
+header {
+  background: linear-gradient(to right, #0f3357, white);
+  background-image: url("../images/header-image.jpg");
+  background-size: cover;
+  color: white;
+  font-weight: bold;
+  text-align: left;
+}
+
+.buy {
+  float: right;
+  width: 5%;
+  margin-top: 2%;
+  margin-right: 20%;
+}
+
+@media screen and (max-width: 768px) {
+  .buy {
+    width: 10%;
+  }
+}
+
+.cart {
+  width: 40%;
+  height: 20%;
+  margin-bottom: 12%;
+}
+
+@media screen and (max-width: 576px) {
+  .cart {
+    width: 60%;
+    font-size: 80%;
+  }
+}
+
+@media screen and (min-width: 577px), (max-width: 768px) {
+  .cart {
+    width: 40%;
+    font-size: 80%;
+  }
+}
+
+.checkout {
+  width: 100%;
+  height: 50%;
+  background-color: #39c639;
+  text-align: center;
+  padding: 5% 20% 5% 20%;
+  color: white;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo {
+  padding-top: 1%;
+  margin-left: 10%;
+  margin-bottom: 15%;
+  color: #39c639;
+  width: 30%;
+  font-size: 110%;
+  font-family: 'Open Sans', sans-serif;
+  font-style: italic;
+}
+
+@media screen and (max-width: 576px) {
+  .logo {
+    width: 45%;
+    font-size: 70%;
+  }
+}
+
+.header_nav {
+  background-color: #f2f2f2;
+  color: #0f3357;
+  padding-left: 15%;
+  padding-top: 1%;
+  padding-bottom: 1%;
+  font-size: 120%;
+}
+
+/*End header style and begin index page style*/
+
+.intro {
+  float: left;
+  width: 40%;
+  margin-left: 15%;
+  margin-top:2%;
+  font-size: 100%;
+}
+
+@media screen and (max-width: 768px) {
+  .intro {
+    width: 60%;
+    margin-left: 5%;
+  }
+}
+
+@media screen and (max-width: 576px) {
+  .intro {
+    width: 90%;
+    margin-left: 20px;
+  }
+}
+
+p {
+  font-family: 'Lato', sans-serif;
+}
+
+.golden {
+  display: block;
+  float: none;
+  width: 100%;
+  padding-right: 5%;
+  float: left;
+  margin-bottom: 5%;
+}
+
+h2 {
+  color: #0f3357;
+  font-size: 200%;
+  font-family: 'PT Serif', serif;
+}
+
+a {
+  color: inherit;
+}
+
+aside {
+  display: block;
+  float: right;
+  width: 20%;
+  margin-right: 15%;
+  font-size: 80%;
+}
+
+.aside_image {
+  width: 80%;
```

Comment:
I would consider changing this to 100% or else edit the margins for the p elements, some text is not lining up with the edges of the images for the aside.